deploy/jenkins-ci/Makefile.operator.jenkins

```diff
@@ -91,25 +91,24 @@ operator-build-release:
 	OPERATOR_QUAY_NAME="$(QUAY_OPERATOR_NAME)" \
 	  $(MAKE) clean build
 
+# prepare operator tags
+QUAY_OPERATOR_TAG = $(QUAY_OPERATOR_NAME):$(VERSION_TAG)
+ifneq ($(IS_SNAPSHOT),y)
+   # Create or update vX.Y tags (without the patch number)
+   # quay.io/kiali-operator
+   QUAY_OPERATOR_TAG := $(QUAY_OPERATOR_TAG) $(QUAY_OPERATOR_NAME):v$(OPERATOR_VERSION_BRANCH)
+endif
+
 operator-push-quay:
 ifdef QUAY_USER
 ifdef QUAY_PASSWORD
 	@echo "Logging in to Quay.io..."
 	@docker login -u "$(QUAY_USER)" -p "$(QUAY_PASSWORD)" quay.io
 endif
 endif
-	OPERATOR_CONTAINER_VERSION="$(VERSION_TAG)" \
+	QUAY_OPERATOR_TAG="$(QUAY_OPERATOR_TAG)" \
 	  OPERATOR_QUAY_NAME="$(QUAY_OPERATOR_NAME)" \
-	 $(MAKE) push
-ifneq ($(IS_SNAPSHOT),y)
-   # Create or update vX.Y tags (without the patch number)
-   # quay.io/kiali-operator
-	docker tag "$(QUAY_OPERATOR_NAME):$(VERSION_TAG)" "$(QUAY_OPERATOR_NAME):v$(OPERATOR_VERSION_BRANCH)"
-	docker push "$(QUAY_OPERATOR_NAME):v$(OPERATOR_VERSION_BRANCH)"
-	docker rmi "$(QUAY_OPERATOR_NAME):v$(OPERATOR_VERSION_BRANCH)"
-endif
-   # Clean-up
```

Comment:
@morlay Same comment: can you do the clean-up?

Reply:
buildx will create container with moby/buildkit.
image layers cache in that container. not in host.
and buildkit will auto gc.

Reply:
OK, I think we can try the garbage collector.

By the way, I see there is a `--pull` flag in the build command. We previously had an issue were cached images never refreshed and we fixed by including the `--pull` flag in `docker build` command. I'm wondering if there would be a similar issue if no `--pull` flag is passed to buildx.

Reply:
I checked my buildx builder. the deps images always be pulled for each build.
but this may because of my buildkit config.

I will add `--pull` to make it safe